docs/wxm-devices/introduction.mdx

```diff
@@ -20,7 +20,7 @@ import useBaseUrl from "@docusaurus/useBaseUrl";
 
 <hr />
 
-In order to join our global weather network and get rewarded in WXM tokens, you need to buy one at our [shop](https://weatherxm.com/shop/).
+In order to join our global weather network and get rewarded in WXM tokens, you need to purchase a bundle from our [shop](https://weatherxm.com/shop/).
```

Comment:
Let's send them only to `weatherxm.com`. Sub-pages may changes, especially the shop page.